app/app/models/users/student.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy.orm import relationship
+
+from app.db.base_class import Base
+from app.models.term import Term
+from app.models.users.user import User
+
+
+class Student(Base):
+    user_id = Column(
+        Integer,
+        ForeignKey(f'{User.__table__.name}.id', ondelete='CASCADE'),
+        index=True,
+        primary_key=True,
+        nullable=False,
+    )
+    user = relationship("User")
+    term_id = Column(Integer, ForeignKey(f'{Term.__table__.name}.id', ondelete='CASCADE'), index=True, nullable=True)
```

Comment:
should we add a relationship to term as well?

Reply:
Also, should we add a backref to User relationship? So that User will also have it's own Student/Professor property

Reply:
Added term relationship, backref will get a bit complex and isn't really needed IMO